src/lib/sw_engine/tvgSwFill.cpp

```diff
@@ -22,7 +22,7 @@
 #include <float.h>
 #include <math.h>
 #include "tvgSwCommon.h"
-
+#include <iostream>
```

Comment:
is it necessary?

Reply:
such an easy pr and so many problems... sorry. I hope now it's ok